.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: 17
```

Comment:
Bump java to java 17. The graphql-simple-bindings dependency is built with java 17.